parm/stage/stage.yaml.j2

```
@@ -0,0 +1,204 @@
+#############################################################
+# This yaml is intended to be of the following structure:
+# key1:
+#     mkdir:
+#         - "COM directory to create"
+#     copy:
+#         - ["source_file", "destination_file"]
+# key2:
+#     mkdir:
+#         - "COM directory to create"
+#     copy:
+#         - ["source_file", "destination_file"]
+#
+# Any number of keys with nested mkdir and copy are permitted
+# Jinja is permitted in this yaml, as long as the keys are:
+# - COMOUT_
+# - DO_ATM, DO_OCN, DO_ICE, etc.
+# For a full list see scripts/exglobal_stage_ic.py
+#############################################################
+
+# Set variables used below
+{% set current_cycle_YMD = current_cycle | to_YMD %}
+{% set current_cycle_HH = current_cycle | strftime("%H") %}
+{% set previous_cycle_YMD = previous_cycle | to_YMD %}
+{% set previous_cycle_HH = previous_cycle | strftime("%H") %}
+{% set m_prefix = model_start_date_current_cycle | to_YMD + "." + model_start_date_current_cycle | strftime("%H") + "0000" %}
```

If `strftime` is being used to format, one can pass the `0000` as part of the format, or one can completely take over `strftime` to do exactly what is needed.  Such as:
```suggestion
{% set m_prefix = model_start_date_current_cycle | strftime("%Y%m%d.%H0000") %}
```

---

Thanks for the feedback! I have incorporated this @ 7ae9ea49.